app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const {
+	handlePsqlErrors,
+	handleCustomErrors,
+	handleServerErrors,
+} = require('./controllers/errors.controller')
+const { getTopics } = require('./controllers/topics.controller')
+
+const app = express()
+app.use(express.json())
+
+app.get('/api/topics', getTopics)
+
+app.use((req, res, next) => handlePsqlErrors)
+app.use((req, res, next) => handleCustomErrors)
+app.use((req, res, next) => handleServerErrors)
```

Comment:
These custom error handlers are not doing anything yet, and are untested.  They are also written in a way that might cause issues.  But as they are untested this might be being missed. Avoid using boilerplate code.  Only write code to produce the functionality you are trying to create through TDD.